base/src/main/java/com/smartdevicelink/streaming/video/VideoStreamingParameters.java

```diff
@@ -75,16 +76,18 @@ public VideoStreamingParameters(){
 	    format = new VideoStreamingFormat();
 	    format.setProtocol(DEFAULT_PROTOCOL);
 	    format.setCodec(DEFAULT_CODEC);
+	    stableFrameRate = true;
     }
 
     public VideoStreamingParameters(int displayDensity, int frameRate, int bitrate, int interval,
-                                    ImageResolution resolution, VideoStreamingFormat format){
+                                    ImageResolution resolution, VideoStreamingFormat format, boolean stableFrameRate){
```

Comment:
This would be considered a breaking change, can you please keep the old constructor and deprecate it and then add in your new constructor.

in the old constructor you can default the stableFrameRate value to be true

Reply:
I have restored previous constructor as you suggested.